app/assets/javascripts/main.js.erb.orig

```diff
@@ -0,0 +1,881 @@
+$(function() {
```

Comment:
Just realized this whole file is the result of merge conflicts, it may be sufficient to just drop this one.